src/init.cpp

```diff
@@ -190,12 +190,20 @@ static void CreateNewConfigFile()
 
     myConfig
         << "addnode=addnode-us-central.cycy.me\n"
+        << "addnode=ch.gridcoin.pl\n"
```

Comment:
The`ch.gridcoin.pl`,`fi.gridcoin.pl`,.. ones don't need to be added because `seed.gridcoin.pl`addnode is a DNS load balancer for those nodes

Reply:
Updated.